fhir-validation/src/main/java/com/ibm/fhir/validation/FHIRValidator.java

```diff
@@ -120,8 +130,11 @@ private FHIRValidator() { }
      * Validate a resource, using an {@link EvaluationContext}, against constraints in the base specification and
      * resource-asserted profile references and/or specific profile references.
      *
-     * <p>Resource-asserted profiles which are unknown to the server will be ignored. Conversely,
-     * unknown profiles passed explicitly as arguments will result in a FHIRValidationException.
+     * <p>Resource-asserted profile references that are not available in the FHIRRegistry result in issues with severity WARNING.
+     *
+     * <p>Unknown profile references passed as arguments to this method result in issues with severity ERROR.
+     *
+     * <p>Profiles that are incompatible with the resource being validated result in issues with severity ERROR.
```

Comment:
```suggestion
     * <p>Profiles that are incompatible with the resource type being validated result in issues with severity ERROR.
```

Reply:
and the same change on all the other variants

Reply:
Updated Javadoc